### PR TITLE
Fix extracting attribute value

### PR DESCRIPTION
**Pull request by tsuwatch**
Prioritize the value specified by content

### CODE REVIEW

**Comment by tkawa** [comment]
I looked into the spec of Microdata, and found out two different definitions of itemprop's value.
- [HTML Microdata by W3C Working Draft 26 April 2018](https://www.w3.org/TR/microdata/#values)
- [HTML Living Standard by WHATWG 13 September 2018](https://html.spec.whatwg.org/multipage/microdata.html#values)

These two specs differ in whether the `content` attribute is allowed for general elements. I prefer somewhat WHATWG, but I want more information to make a decision.
Do you have any good idea?

File: test/data/example.html [hunk 1]
```diff
@@ -16,7 +16,8 @@
           <span itemprop="name">
             <a itemprop="url" href="http://www.lib.ncsu.edu">NCSU Libraries</a>
           </span>
-        </span>.
+        </span>. Born in 
+        <span itemprop="birthDate" content="1900-09-14T21:30">Sep 14</span>
```
Comment (tkawa) [inline test/data/example.html]: Also `<time itemprop="birthDate" datetime="1900-09-14T21:30">Sep 14</time>` is good, if that helps.

**Comment by tsuwatch** [comment]
Use `content` attribute in `div` element in Example 2 https://schema.org/Person
I would like to separate metadata from display data.

**Comment by tkawa** [comment]
I got it. I recommend using `time` element instead of `div`/`span` if you want to express a time.
Anyway, this PR is useful. Thank you!

**Comment by tsuwatch** [comment]
Thank you for merging!

**Pull request merged**